diagnostic.md

```diff
@@ -56,8 +73,8 @@ let answer = 'not empty';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!");
+  if (answer !== 1 >= <= 10) {
+   console.log("Guess between 1 and 10");
    break;
   }
 }
```

Comment:
This should be in bin/diagnostic.js.  Check the solution branch if you want to see a solution to this problem.